app/helpers.php

```diff
@@ -170,17 +178,21 @@ function get_signed_language_for_written_language(string $code): string
      */
     function get_language_exonym(string $code, string $locale = '', bool $capitalize = true): null|string
     {
-        if ($locale === '') {
-            $locale = locale();
-        }
+        $locale = $locale === '' ? locale() : $locale;
```

Comment:
Minor but might be easier to make $locale nullable and default to `null`. Then here you could do the follow:

```suggestion
        $locale ??= locale();
```

Or if you can use `locale()` as the default in the parameters you can skip this line entirely.